frontend/packages/dev-console/integration-tests/testData/git-import-flow.data.ts

```diff
@@ -0,0 +1,15 @@
+export const testData = {
```

Comment:
Rather than creating such file, we can create the parameterized functions which accepts data when we call the functions.

Reply:
This is test data file, which we will pass in scenario files where ever required

Reply:
If we create a parametarized function, we won't need a testData file.